keyboards/capsunlocked/cu80/v2_ansi/readme.md

```diff
@@ -0,0 +1,15 @@
+# CU80 v2 ANSI
+
+The CU80 v2 ANSI is the PCB for round 2 of the CU80 TKL keyboard, with fixed layout ANSI and VIA support.
```

Comment:
Can you add picture about the board? A link to an imgur picture or something.
(Same goes for the ISO version)

Reply:
Done!